web/css/completeMeeting.opentok.css

```diff
@@ -0,0 +1,117 @@
+html {
+  background-color: #131415;
+  mix-blend-mode: normal;
+  background-image: url("/images/vonage-v-logo-background.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 100%;
+  height: 100%;
+}
+.completeMeeting {
+  margin-left: 100px;
+  font-family: "Spezia", arial !important;
+  color: white;
+}
+@media only screen and (max-width: 480px), only screen and (-webkit-min-device-pixel-ratio: 1.5, -o-min-device-pixel-ratio: 3/2, min--moz-device-pixel-ratio: 1.5, min-device-pixel-ratio: 1.5) only screen and (orientation:portrait) and (max-width: 480px), only screen and (orientation:portrait) and (-webkit-min-device-pixel-ratio: 1.5, -o-min-device-pixel-ratio: 3/2, min--moz-device-pixel-ratio: 1.5, min-device-pixel-ratio: 1.5) {
+  .completeMeeting {
+    margin-left: 75px;
+  }
+}
+.completeMeeting .completeMeeting-box-container {
+  display: -ms-inline-flexbox;
+  display: -ms-flexbox;
+  display: flex; 
+  -ms-flex-align: start; 
+  align-items: flex-start;
+  -ms-flex-direction: row;
+  flex-direction: row;
+}
+
+@media only screen and (max-width: 480px), only screen and (-webkit-min-device-pixel-ratio: 1.5, -o-min-device-pixel-ratio: 3/2, min--moz-device-pixel-ratio: 1.5, min-device-pixel-ratio: 1.5) only screen and (orientation:portrait) and (max-width: 480px), only screen and (orientation:portrait) and (-webkit-min-device-pixel-ratio: 1.5, -o-min-device-pixel-ratio: 3/2, min--moz-device-pixel-ratio: 1.5, min-device-pixel-ratio: 1.5) {
+  .completeMeeting .completeMeeting-box-container {
+    -ms-flex-direction: column;
+    flex-direction: column;
```

Comment:
The margins look a bit off in mobile. Maybe add this to `.completeMeeting`:

```
margin-left: 24px;
margin-right: 32px;
```

Also, the space above and below the "Start Building for Free" button is a bit off on mobile.